### DISCUSSION TITLE
API documentation feature

### DISCUSSION THREAD

**Discussion by haurs** [hoppscotch/hoppscotch#3429] [Q&A]
Hi! I wonder if there's any existing feature for documenting the API we setup in Hoppscotch? Postman for example has a feature for this but I can't find any place todo the same in Hoppscotch.

Any recommendations would be much appreciated!

**Comment by calebsmithdev** [comment]
I'd love for support for this. Either that or comments - just something so I can leave a note for what the API is and provide context to my team.

**Reply by liyasthomas** [reply]
We've plans to introduce API documentation support in 2024. Thanks for the feedback.

**Reply by calebsmithdev** [reply]
That's fantastic! Thank you so much :)

**Reply by JunaidMancheri** [reply]
i would love to collaborate
